actors.md

```diff
@@ -323,19 +318,10 @@ type StorageMiner struct {
 	// to this miner
 	PeerID peer.ID
 
-	// PledgeBytes is the amount of space being offered by this miner to the network
-	PledgeBytes BytesAmount
```

Comment:
I know there are some ideas in the running for our collateral function that are not as simple as `collateral = C * PledgeBytes`.  Are all of these candidate collateral functions invertible?  If not user experience will suffer because it would be ambiguous how many bytes a miner has left to store.

I'm aware that we are not actually enforcing the pledge amount so malicious miners could always look available and not accept any requests, but the system shouldn't make it hard for clients to know if honest miners have room left.  Is the plan to move this signal off the blockchain ?

Reply:
If we do not track storage and consensus collateral separately then I think ambiguity is always there.

Reply:
The specification shows that the storage miner actor [does track consensus collateral (`ConsensusCollateral`)](https://github.com/filecoin-project/specs/pull/312/files#diff-8cd5872b87e5b64500c99ea50b3a9241R634). 

`ActiveCollateral` is [divided into storage collateral and pledge collateral](https://filecoinproject.slack.com/archives/CHMNDCK9P/p1559239486004600?thread_ts=1559235221.001500&cid=CHMNDCK9P).

It is also the case that a storage miner actor's FIL balance [is always less than or equal](https://filecoinproject.slack.com/archives/CHMNDCK9P/p1559237847003700?thread_ts=1559235221.001500&cid=CHMNDCK9P) to `ActiveCollateral` plus `ConsensusCollateral`.

Reply:
the construction of the collateral function is separate from the issue of not being able to prove how much storage a given miner has available.

We haven't yet separated pledge collateral and storage collateral, that will happen soon